.github/CONTRIBUTING.md

```diff
@@ -84,18 +84,31 @@ There are three different consumption strategies you can utilize
 
 * Using a release version of terraform
 
+Note that these instructions apply to `0.13+`. For prior Terraform versions, look at past versions of this page for instructions.
```

Comment:
Nit: Should we bold this or something? I guess it will matter less in the future

Reply:
🤷‍♀️ I don't have strong feelings either way, chose not to. I wish `~>` for a note was standard in markdown.